packages/macros/cw-orch-fns-derive/src/helpers.rs

```diff
@@ -116,26 +91,79 @@ pub(crate) fn is_type_using_into(field_type: &Type) -> bool {
         _ => false,
     }
 }
+#[derive(Default)]
+pub struct EnumAttributes {
+    pub disable_fields_sorting: bool,
+}
 
-pub(crate) fn has_cw_orch_attribute(attrs: &[Attribute], attribute_name: &str) -> bool {
-    for attr in attrs {
-        if attr.path.segments.len() == 1 && attr.path.segments[0].ident == "cw_orch" {
-            // We check the payable attribute is in there
-            for token_tree in attr.tokens.clone() {
-                if let TokenTree::Group(e) = token_tree {
-                    for ident in e.stream() {
-                        if ident.to_string() == attribute_name {
-                            return true;
-                        }
-                    }
+pub(crate) fn parse_enum_attributes(item_enum: &ItemEnum) -> EnumAttributes {
+    let mut enum_attributes = EnumAttributes::default();
+    for attr in &item_enum.attrs {
+        if attr.path().is_ident("cw_orch") {
```

Comment:
So in syn2 the namespacing of macros is refered to as the "ident"?

Reply:
Yes ! This is directly inspired from the official syn2 docs